.github/workflows/lint.yml

```diff
@@ -0,0 +1,13 @@
+name: Lint
+
+on: push
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
Prefer to add a line break between each step. Also, a bit of inconsistency with `test.yml` as name this step has no `name` attribute which I think is fine by the way.

Reply:
Fixed in d3861b92c960ab808b74e0869f5d30d9d4040346 9106033b6f5373274fc0787d66bfabfb20d0abd1